Assets/BossRoom/Scripts/Client/UI/IPUIMediator.cs

```diff
@@ -87,9 +75,14 @@ public void HostIPRequest(string ip, string port)
 
             m_GameNetPortal.PlayerName = m_PlayerNameLabel.text;
 
-            m_GameNetPortal.StartHost(ip, portNum);
-
-            m_SignInSpinner.SetActive(true);
+            if (m_GameNetPortal.StartHost(ip, portNum))
+            {
+                m_SignInSpinner.SetActive(true);
+            }
+            else
+            {
+                PopupManager.ShowPopupPanel("Connection Failed", "Server failed to bind.");
```

Comment:
can a different issue cause StartHost to return false?

Reply:
Also, while testing I noticed that if we get here after failing to bind, further StartHost and StartClient calls all fail. It looks like a bug for the sdk though

Reply:
How do we bubble issue details SDK side? error logs?
Also bug that bug!